defaults/main.yml

```diff
@@ -52,3 +52,9 @@ openwisp2_topology_save_snapshot_frequency:
     day: "*"
     hour: "23"
     minute: "30"
+openwisp2_nginx_ssl_config:
+  gzip: "on"
+  gzip_comp_level: "6"
+  gzip_proxied: "any"
+  gzip_min_length: "1000"
+  gzip_types: "text/plain text/html image/svg+xml application/json application/javascript text/xml text/css application/xml application/x-font-ttf font/opentype"
```

Comment:
can you turn this long string into a list of items that is then reassembled in the nginx template?

Eg: in the template you can probably use `{{ ' '.join(gzip_types) }}`